src/bios.rs

```diff
@@ -81,6 +93,38 @@ impl Bios {
         }
         Ok(())
     }
+
+    // check bios_boot partition on gpt type disk
+    fn get_bios_boot_partition(&self) -> Result<Option<String>> {
+        let target = self.get_device()?;
+        // lsblk to list children with bios_boot
+        let output = Command::new("lsblk")
```

Comment:
Side note if we fold into bootc, we already have the equivalent of this code there https://github.com/containers/bootc/blob/a9cfdc83cc747ac97a6f396735e8d6cb57c68e30/lib/src/blockdev.rs#L281

Reply:
Good insight, hmm, bootupd can not use that code, need to find workaround for this.

Reply:
I'm thinking recently again about https://github.com/coreos/bootupd/issues/432 which would fix this.

Anyways, it was just an aside.

In the short term maybe we can try to literally copy in `blockdev.rs` from bootc or so.

Reply:
Maybe can try with `/dev/disk/by-partlabel/BIOS-BOOT`, but this does not exist on silverblue (start using BIOS):
```
$ sudo blkid
/dev/vda1: PARTUUID="928fc854-1795-4856-b9d7-7818dff9bfbc"
$ ls /dev/disk/by-partlabel/
ls: cannot access '/dev/disk/by-partlabel/': No such file or directory
```

Reply:
> I'm thinking recently again about https://github.com/coreos/bootupd/issues/432 which would fix this.

No objection, maybe can add the check when call bootupd.

> In the short term maybe we can try to literally copy in blockdev.rs from bootc or so.

Will try this, thanks for the pointer.